test/core-flags/.gitignore

```diff
@@ -0,0 +1,3 @@
+/test
+/bin
+/dist
```

Comment:
We don't need this once rebased with master

Reply:
removed :+1: